src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -90,8 +90,11 @@ public static int sqlite3_bind_parameter_index(Sqlite3StmtHandle pStmt, string z
             }
         }
 
-        [DllImport("sqlite3", CallingConvention = CallingConvention.Cdecl, CharSet = CharSet.Unicode, ExactSpelling = true)]
-        public static extern int sqlite3_bind_text16(Sqlite3StmtHandle pStmt, int i, string zData, int n, IntPtr xDel);
+        [DllImport("sqlite3", CallingConvention = CallingConvention.Cdecl, ExactSpelling = true)]
+        private static extern int sqlite3_bind_text(Sqlite3StmtHandle pStmt, int i, IntPtr zData, int n, IntPtr xDel);
+
+        public static int sqlite3_bind_text(Sqlite3StmtHandle pStmt, int i, string data, int n, IntPtr xDel) =>
+            sqlite3_bind_text(pStmt, i, MarshalEx.StringToHGlobalUTF8(data), n, xDel);
```

Comment:
See [the previous implementation](https://github.com/aspnet/Microsoft.Data.Sqlite/blob/ff35414ba1d88340bbdc195c2306a97b10d52a6d/src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs#L70-L88).